packages/lisk-api-client/src/api_resource.ts

```diff
@@ -67,26 +67,26 @@ export class APIResource {
 		req: AxiosRequestConfig,
 		retry: boolean,
 		retryCount: number = 1,
-	): Promise<ApiResponse> {
+	): Promise<APIResponse> {
 		const request = Axios.request(req)
 			.then((res: AxiosResponse) => res.data)
 			.catch(
-				(error: AxiosError): Error => {
+				(error: AxiosError): void => {
 					if (error.response) {
-						if (error.response.data && error.response.data.message) {
+						const { status } = error.response;
+						if (error.response.data) {
+							const {
+								error: errorString,
```

Comment:
where is errorString defined?

Reply:
this is the definition of `errorString`.
It's destructure with giving different name